packages/@tailwindcss-cli/src/commands/build/index.ts

```diff
@@ -255,6 +266,44 @@ export async function handle(args: Result<ReturnType<typeof options>>) {
   }
 }
 
+async function createWatchers(globs: GlobEntry[], handle: (files: string[]) => void) {
+  let watchers = disposables()
+  let files = new Set<string>()
+
+  let d = disposables()
+  function flush() {
+    d.dispose()
+    d.queueMacrotask(() => {
+      handle(Array.from(files))
```

Comment:
If we have two globs yielding the same file, the idea here is that we only emit the files once, right?

I think we should check if files is non-empty before we call handle otherwise we will still have two callback invocations (even though only one contains the changed/added file)

Reply:
Nevermind, this is what disposables is for and why we call `dispose()` before queueing another macrotask. This will cancel all previously scheduled macro tasks.